.codecov.yml

```
@@ -2,7 +2,7 @@
 
 codecov:
   notify:
-    after_n_builds: 22  # The number of test matrix+lint jobs uploading coverage
+    after_n_builds: 18  # The number of test matrix+lint jobs uploading coverage
```

Looks like it actually uploads more: https://app.codecov.io/github/aio-libs/yarl/commit/a051f9f5d0d6a7bbd9f6793436c86b019c7e8aba.
```suggestion
    after_n_builds: 23  # The number of test matrix+lint jobs uploading coverage
```